syntax = "proto3";

message ContouringOutput {
    string value = 1;
}

message ContouringEmpty{}

service ContourServices{
    rpc computeContour(ContouringEmpty) returns (ContouringOutput) {}
}